packages/components/src/components/custom-select/custom-select.lite.tsx

```diff
@@ -701,12 +701,12 @@ export default function DBCustomSelect(props: DBCustomSelectProps) {
 
 	// If we inform the consumer we don't want to trigger the onOptionSelected event again
 	onUpdate(() => {
-		if (
-			props.values &&
-			Array.isArray(props.values) &&
-			props.values !== state._values
-		) {
-			state._values = props.values ?? [];
+		const v = props.values;
+
+		if (Array.isArray(v)) {
+			if (state._values !== v) state._values = v;
+		} else if (v == null) {
```

Comment:
```suggestion
		if (Array.isArray(props.values)) {
			if (state._values !== props.values) state._values = props.values;
		} else if (props.values == null) {
```

I don't know what this abstraction would be good for except loosing clarity.